Update _index.md (#18157)
DataDog/documentation: content/en/serverless/google_cloud_run/_index.md

ENV DD_ENV=datadog-demo
ENV DD_VERSION=1

> # this env var is needed for trace propagation to work properly in cloud run.

Reviewer: ```suggestion
# this env var is needed for trace propagation to work properly in Cloud Run.
```

> # make sure to also set this variable for all Datadog instrumented downstream services.

Reviewer: ```suggestion
# ensure to set this variable for all Datadog-instrumented downstream services.
```

Reviewer: Leaving just these suggestions here to avoid duplication

Author: fixed!

- **Custom Metrics:** You can submit custom metrics using a [DogStatsd client][4]. For monitoring Cloud Run and other serverless applications, use [distribution][9] metrics. Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][11]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.

> - **Trace Propagation:** In order to propagate trace context for distributed tracing, set the `DD_TRACE_PROPAGATION_STYLE` environment variable to `'datadog'` for your Cloud Run app and any Datadog instrumented services downstream of it.

Reviewer: ```suggestion
- **Trace Propagation:** In order to propagate trace context for distributed tracing, set the `DD_TRACE_PROPAGATION_STYLE` environment variable to `'datadog'` for your Cloud Run app and any Datadog-instrumented services downstream of it.
```

   const provider = new NodeTracerProvider({
      resource: new Resource({

>           [ SemanticResourceAttributes.SERVICE_NAME ]: 'your-service-name',

Reviewer: ```suggestion
          [ SemanticResourceAttributes.SERVICE_NAME ]: '<your-service-name>',
```
